elastic/kibana
# [Infra UI] Make queries more robust against missing indices

## Summary

This fixes several queries that failed when indices didn't exist and were not specified using a wildcard at the same time. The problem was first noticed by a community member on the node detail page:

![grafik](https://user-images.githubusercontent.com/973741/59509929-b300ae80-8eb2-11e9-9f49-794fe1d6189f.png)

## Testing

1. Ingest host metrics.
2. Configure the "logs indices" setting in the source configuration such, that it doesn't match any index and doesn't contain a wildcard, e.g. `this-index-does-not-exist`.
3. Visit the node detail page for a host.
4. Observe that the page is shown as expected as opposed to before this fix, when the error depicted above would be shown.
